Sources/TarkaUI/Components/Attachment/TUIAttachmentUpload.swift

```diff
@@ -113,6 +119,42 @@ public struct TUIAttachmentUpload: View {
 
 public extension TUIAttachmentUpload {
   
+  struct InputItem {
```

Comment:
I feel the model and it's styling attributes such as size and color should be in separate objects

Reply:
@arvindhsukumar if we have more objects, it makes some confusions, shall we name some thing else?

Reply:
Sure.. Anything other than InputItem, which to me indicates a model object, not styling properties

Reply:
@arvindhsukumar Updated